neo/Prompt/Commands/SC.py

```diff
@@ -283,7 +284,25 @@ def execute(self, arguments):
                 print("Incorrect password")
                 return False
 
-            return InvokeContract(wallet, tx, Fixed8.Zero(), from_addr=from_addr)
+            n_fee = Fixed8.Zero()
+            if priority_fee is not None:
+                n_fee = priority_fee
+                if n_fee is False:
+                    logger.debug("invalid fee")
+                    return False

```

Comment:
2 things
1) if the user does not have logging set to debugging then their deploy will fail silently after passw entry
2) we should capture invalid arguments before test running.

Reply:
Answer 1) This logic follows the same logic as in `Send.py` and you can see here https://github.com/CityOfZion/neo-python/blob/6c1d68f45af2fc0909cb64cfd8c51dab2dcfd195/neo/Prompt/Commands/tests/test_sc_commands.py#L286-L302 that a bad fee will present a normal printout.

Answer 2) I can move the code up in the sequence 👍